backend/tests/conftest.py

```
@@ -0,0 +1,83 @@
+fake_filters = types.ModuleType("Generator.question_filters")
+
+# -------------------------------------------------
+# Fake classes used by backend/server.py
+# -------------------------------------------------
+
+class FakeMCQGenerator:
+    def generate_mcq(self, *args, **kwargs):
+        return {"questions": []}
+
+class FakeFileProcessor:
+    def process(self, *args, **kwargs):
```

Ruff (0.15.2)

[warning] Unused method argument: `args`

(ARG002)